src/main/webapp/assets/css/custom_style.css

```diff
@@ -104,6 +104,15 @@ li.activeCourse button.setSections {
     height: 30px;
     width: 30px;
 }
+.spinner2{
+    background: url("../images/spinner-xs.gif") no-repeat scroll 0 0 transparent;
+    display: inline-block;
+    height: 30px;
+    width: 30px;
+    line-height: 1.42857;
+    margin-bottom: 1px;
+    vertical-align: middle;
+}
```

Comment:
The other one is on the first search and is controlled via Angular. A bit of duplication, but unavoidable till we angularize the whole thing if there is to be a second phase.